CONTRIBUTING.md

```diff
@@ -39,9 +38,20 @@ npx react-native start --watchFolders /path/to/cloned/cli/
 npx react-native run-android
 ```
 
+Once you're done with testing and you'd like to get back to regular setup, run `yarn unlink` instead of `yarn link` from above command. Then `yarn install --force`.
+
+If you use `yarn` v2, run
+```sh
+cd /my/new/react-native/project/
+yarn link /path/to/cloned/cli/ --all
+
+npx react-native start --watchFolders /path/to/cloned/cli/
+npx react-native run-android
```

Comment:
These lines are copied over from 37-38. Let's split linking and running to separate code blocks to avoid duplication. This way the note on line 53 will be relevant for both instructions – right now people running yarn v1 may miss it, because it's below yarn v2 section